src/scss/components/checkbox/_index.scss

```diff
@@ -125,17 +126,22 @@ hx-checkbox-control {
 
     &:checked,
     &:indeterminate {
+      color: $blue-700;
       color: var(--hxCheckbox-indeterminate-color, $blue-700);
 
       ~ label > hx-checkbox {
+        background-color: $blue-700;
```

Comment:
@100stacks so i'm just asking in order to gain knowledge. I see we sometimes duplicate properties. So in this case we have the color variable here, and then the next line we have the same property, but with the CSS variable. This is required for browser compatibility and fallback, correct? So if the browser doesn't support CSS variables, then it will at least have the usual color var, correct?

Reply:
@ssalinas24, yes that's correct.  We're using duplicate properties for IE11 support fallback.  It's cleaner and simpler than using a library to provide to polyfill IE11 CSS variables/custom property support.